server/api/product.ts

```diff
@@ -1,15 +1,15 @@
 import { getGQLClient } from "../util"
-import type { KiboIncomingMessage, Response } from "../types"
+import type { KiboIncomingMessage, Response } from "../types/Api"
 import { productDetailQuery } from "../queries"
 
 export default async function getProduct(
```

Comment:
Apply the return type

Reply:
may be this is a dummy file, i am not taking care of